### DISCUSSION TITLE
[Question] Is this app ever going to work without a reset again?

### DISCUSSION THREAD

**Discussion by TairikuOokami** [2fast-team/2fast#187] [Q&A]
I use this app for years, it worked with Windows Hello till 24H2.
I have to reset it every time to launch it and to use the password.
I use it as a backup, so I do not care much, just wondering, why?

**Comment by jp-weber** [comment]
You mean you reset the app, because Windows Hello is not working since 24H2?

**Reply by TairikuOokami** [reply]
Yes, it did not recognize Windows Hello upon restart, as mentioned in other issues.
But lately it does not even ask to use Hello anymore upon restart, enabled in settings.

**Reply by jp-weber** [reply]
This sounds very much like a problem with Windows Hello, because the app only queries the availability of Windows Hello and, if available, executes this. Have you tried reset Windows Hello under Win11 24h2?
